packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -29,13 +32,26 @@ const emit = defineEmits<{
    * Emitted when the current search input value changes.
    */
   "update:searchTerm": [searchTerm: string];
+  "update:open": [open: boolean];
   /**
    * Emitted if lazy loading is triggered / the users scrolls to the end of the options.
    * See property `lazyLoading` for enabling the lazy loading.
    */
   lazyLoad: [];
 }>();
 
+const searchTerm = useManagedState(
+  toRef(() => props.searchTerm),
```

Comment:
Is there a reason why toRef instead of computed?

Reply:
for this use-case, there doesn't seem to be a funtional difference between the two.
Looking at the respective implementations `toRef` seems to have less overhead.
Also it seems to be the recommended way:
`toRef() is useful when you want to pass the ref of a prop to a composable function` (https://vuejs.org/api/reactivity-utilities.html#toref)